src/app/util/af-types.h

```diff
@@ -63,6 +57,21 @@ typedef void (*EmberAfGenericClusterFunction)(void);
  */
 #define MATTER_DM_NULL_MANUFACTURER_CODE 0x0000
 
+// The following define names are relevant to the ZAP_CLUSTER_MASK macro.
+#define MATTER_CLUSTER_BITMAP_INIT_FUNCTION 0x01
+#define MATTER_CLUSTER_BITMAP_ATTRIBUTE_CHANGED_FUNCTION 0x02
+// Bit 2 (0x04)  and Bit3 (0x08) are free.
+#define MATTER_CLUSTER_BITMAP_SHUTDOWN_FUNCTION 0x10
+#define MATTER_CLUSTER_BITMAP_PRE_ATTRIBUTE_CHANGED_FUNCTION 0x20
+#define MATTER_CLUSTER_BITMAP_SERVER 0x40
+#define MATTER_CLUSTER_BITMAP_CLIENT 0x80
```

Comment:
For all of these, and the attribute flags too, I would suggest using FLAG instead of BITMAP in the name.